gradio/pipelines_utils.py

```diff
@@ -66,7 +59,7 @@ def is_transformers_pipeline_type(pipeline, class_name: str):
             ),
             "outputs": components.Label(label="Classification", render=False),
             "preprocess": lambda i: {"images": i},
-            "postprocess": lambda r: {i["label"].split(", ")[0]: i["score"] for i in r},
```

Comment:
This transforms a label name like "tiger, Panthera tigris" into just "tiger", which looks not good.

On `main`:
![CleanShot 2024-04-19 at 16 27 16@2x](https://github.com/gradio-app/gradio/assets/3135397/df06100e-532c-4d27-8303-238bad5d838f)

This change:
![CleanShot 2024-04-19 at 16 27 50@2x](https://github.com/gradio-app/gradio/assets/3135397/71b7b787-8609-403f-8063-bfb2cf7cdae9)

Sample code:
```python
import gradio as gr
from transformers import pipeline


pipe = pipeline("image-classification", "google/vit-base-patch16-224")

demo = gr.Interface.from_pipeline(pipe)

demo.launch()
```

Reply:
Yeah in the general case better to leave the labels as is